matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/OneTimeKeysUploader.kt

```diff
@@ -120,19 +124,22 @@ internal class OneTimeKeysUploader @Inject constructor(
      * @param keyLimit the limit
      * @return the number of uploaded keys
      */
-    private suspend fun uploadOTK(keyCount: Int, keyLimit: Int): Int {
+    private suspend fun uploadOTK(keyCount: Int, keyLimit: Int, shouldGenerateFallbackKey: Boolean): Int {
```

Comment:
This is a bit strange to send fall back key using the recursive method which send one time key. I do not see any mistake, but it's a bit confusing.

Reply:
It was an easy solution for me. Couldn't see alternatives without refactoring multiple functions, but will check again.